SConstruct

```diff
@@ -119,6 +119,8 @@ opts.Add(BoolVariable("tools", "Build the tools (a.k.a. the Godot editor)", True
 opts.Add(EnumVariable("target", "Compilation target", "debug", ("debug", "release_debug", "release")))
 opts.Add("arch", "Platform-dependent architecture (arm/arm64/x86/x64/mips/...)", "")
 opts.Add(EnumVariable("bits", "Target platform bits", "default", ("default", "32", "64")))
+# Add "64" when double support is completed.
```

Comment:
I guess you could add `64` now so that one can *attempt* a 64-bit build, even if not fully supported yet?

Reply:
Done. I didn't want to get hopes up, but yes we may as well put it in now. It won't work *right now* but hopefully soon.